Makefile

```diff
@@ -133,7 +133,7 @@ test-integration integration func: deps test-db-drop test-db-create
 	@echo "=               Running integration tests...             ="
 	@echo "-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
 	@echo
-	@env MY_IP=${MY_IP} ginkgo -r --randomizeAllSpecs --randomizeSuites --skip="\[Integration\].*" .
+	@env MY_IP=${MY_IP} ginkgo -r -tags=integration --randomizeAllSpecs --randomizeSuites --skip="\[Integration\].*" .
```

Comment:
We're still skipping integration testes?

Reply:
Yep. Since the are other flaky tests, I thought about fixing them in another PR.